CONTRIBUTING.md

```diff
@@ -331,93 +330,6 @@ satisfy the tests provided for each problem.
 You can include advice and helpful links for your language track in the
 `SETUP.md` file.
 
-## Maintaining a Track
```

Comment:
It would be a great help if you specified which information you identified as missing.